tools/custom-checks/check-apache-version.ts

```diff
@@ -0,0 +1,81 @@
+   * * package dependency version declarations.   * If you omit this (optional) parameter then the root package.json file
+   * * will be parsed to obtain its value at runtime.   */ readonly version?: string;
+}
+/** * Verifies that each sibling dependency is up to date with the latest version * that was released at any given time.
+ * * Note: This only checks dependency versions for the packages that are hosted
+ * * within this monorepo (Hyperledger Cactus) not dependencies in general. *
+ * * For example if the cmd-api-server package depends on the common package and
+ * * currently the project is on release v1.2.3 then the dependency declaration of
+ * * the package.json file of the cmd-api-server package should also use this
+ * * v1.2.3 version not something outdated such as v0.3.1 because that may cause
+ * * (according to our experience) strange build issues that confuse people a lot
+ * * for no good reason. *
+ * * @returns An array with the first item being a boolean indicating * 1) success (`true`) or 2) failure (`false`) */
+export async function checkPkgLicense(
+  req: ICheckSiblingDepVersionConsistencyRequest,
+): Promise<[boolean, string[]]> {
+  const TAG = "[tools/checkapache-version.ts]";
+  const __filename = fileURLToPath(import.meta.url);
+  const __dirname = path.dirname(__filename);
+  const SCRIPT_DIR = __dirname;
+  const PROJECT_DIR = path.join(SCRIPT_DIR, "../../");
+  const correctLicense = "Apache-2.0";
+  console.log(`${TAG} SCRIPT_DIR=${SCRIPT_DIR}`);
+  console.log(`${TAG} PROJECT_DIR=${PROJECT_DIR}`);
+  if (!req) {
+    throw new RuntimeError(`req parameter cannot be falsy.`);
+  }
+  if (!req.argv) {
+    throw new RuntimeError(`req.argv cannot be falsy.`);
+  }
+  if (!req.env) {
+    throw new RuntimeError(`req.env cannot be falsy.`);
+  }
+  const globbyOpts: GlobbyOptions = {
+    cwd: PROJECT_DIR,
+    ignore: ["**/node_modules"],
+  };
+  const DEFAULT_GLOB = "**/package.json";
+  const pkgJsonPaths = await globby(DEFAULT_GLOB, globbyOpts);
+  const oasPaths = await globby(DEFAULT_GLOB, globbyOpts);
+  console.log(`openapi.json paths (${oasPaths.length})`);
+  console.log(`${TAG} Correct License: ${correctLicense}`);
+  const errors: string[] = [];
+  const passed: string[] = [];
+  const checks = pkgJsonPaths.map(async (pathRel) => {
+    const filePathAbs = path.join(PROJECT_DIR, pathRel);
+    const pkgJson: unknown = await fs.readJSON(filePathAbs);
+    if (typeof pkgJson !== "object") {
+      errors.push(`ERROR: ${pathRel} package.json cannot be empty.`);
+      return;
+    }
+    if (!pkgJson) {
+      errors.push(`ERROR: ${pathRel} package.json cannot be empty.`);
+      return;
+    }
+    if (!isStdLibRecord(pkgJson)) {
+      return;
+    }
+    const { license } = pkgJson;
+    if (license == correctLicense) {
+      const msg = `${TAG} PASSED: ${pathRel} has the correct ${license}. Expected ${correctLicense}`;
+      passed.push(msg);
```

Comment:
@micoferdinand98 This doesn't seem to be going anywhere so I'd just remove the 'passed' messages and the code associated with it.

Reply:
This is already resolve Peter and ready for review again.

![image](https://github.com/hyperledger/cacti/assets/55821921/2da15c1c-b4d5-4692-b95d-6851a6ef98bf)